packages/docs/site/docs/08-query-api/01-index.md

```diff
@@ -33,11 +33,19 @@ You can go ahead and try it out. The Playground will automatically install the t
 | `url`                  | `/wp-admin/`                                    | Load the specified initial page displaying WordPress                                                                                                                                                                                                                                                                                                           |
 | `mode`                 | `seamless`, `browser`, or `browser-full-screen` | Displays WordPress on a full-page or wraps it in a browser UI                                                                                                                                                                                                                                                                                                  |
 | `lazy`                 |                                                 | Defer loading the Playground assets until someone clicks on the "Run" button                                                                                                                                                                                                                                                                                   |
-| `login`                | `1`                                             | Logs the user in as an admin                                                                                                                                                                                                                                                                                                                                   |
+| `login`                | `1`                                             | Logs the user in as an admin. Set to `0` to not log in.                                                                                                                                                                                                                                                                                                        |
```

Comment:
Good catch, thank you @eliot-akira! I know it's not directly related to this PR, but it's such a small and useful change – let's keep it.